packages/flutter/test/material/radio_list_tile_test.dart

```diff
@@ -1588,4 +1588,46 @@ void main() {
     final Offset offsetPlatform = tester.getTopLeft(platform);
     expect(offsetPlatform, const Offset(72.0, 16.0));
   });
+
+  testWidgets('RadioListTile renders with default scale', (WidgetTester tester) async {
+    await tester.pumpWidget(const MaterialApp(
+      home: Material(
+        child: RadioListTile<bool>(
+          value: false,
+          groupValue: false,
+          onChanged: null,
+        ),
+      ),
+    ));
+
+    final Finder widget = find.ancestor(
+      of: find.byType(Radio<bool>),
+      matching: find.byType(Transform),
+    );
+
+    expect(widget, findsNothing);
```

Comment:
Generally, `widget` is expected to have the Widget type; might as well change the name here:

```suggestion
    final Finder transformFinder = find.ancestor(
      of: find.byType(Radio<bool>),
      matching: find.byType(Transform),
    );

    expect(transformFinder, findsNothing);
```